terraform/delta_logs_db.tf

```diff
@@ -0,0 +1,29 @@
+resource "aws_dynamodb_table" "delta-dynamodb-table" {
+    name         = "${local.short_prefix}-delta-logs"
```

Comment:
This seems to be the only usage of "log" in reference to the delta. Could you remove "logs" from this and the file name just so it's consistent.